joomla/joomla-cms
# Add script version to all js files and replace deprecated script signatures usage

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/9638 (js part).
### Summary of Changes

Following https://github.com/joomla/joomla-cms/pull/11289 merge, this PR replaces all deprecated script  methods signature usage across the CMS.

Also:
- adds script version to all js files to solve https://github.com/joomla/joomla-cms/issues/9638
- use the api in conditional ie statements (html5.js)
### Testing Instructions
1. Code review.
2. Apply patch and navigate in the frontend/backend. Confirm all is working fine and check all external js files are loaded and with script version
### Documentation Changes Required

None.

pinging @dgt41
